nodejs/week2/homework/meal-sharing/src/backend/api/reviews-router.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const router = express.Router();
+
+const reviews = require("./../data/reviews.json");
+
+router.get("/", async(request, response) => {
+    try {
+        response.json(reviews);
+        console.log("in/api/reviews");
+    } catch (error) {
+        throw error;
+    }
+});
+//Respond with the json for the review with the corresponding id 
+router.get("/:id", async(req, res) => {
+    try {
+        const { id } = req.params;
+        const reviewId = reviews.filter((review) => review.id === parseInt(id));
+        if (reviewId < 1) {
```

Comment:
Operator '<' cannot be applied to types '{ id: number; numberOfStars: number; content: string; mealId: number; createdAt: number; }[]' and 'number'.ts(2365)